xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-api/src/main/java/org/xwiki/notifications/internal/email/AbstractMimeMessageIterator.java

```diff
@@ -151,9 +139,7 @@ private void computeNext()
             try {
                 // TODO: in a next version, it will be import to paginate these results and to send several emails
                 // if there is too much content
-                this.currentEvents = notificationManager.getEvents(serializer.serialize(currentUser),
-                        NotificationFormat.EMAIL, false, Integer.MAX_VALUE / 4, null,
-                        lastTrigger, Collections.emptyList());
+                this.currentEvents = retrieveCompositeEventList(currentUser);
```

Comment:
Live notif, I don't think so. But periodic, we will. But forget that, your code is internal so it could be changed afterwards.